bigbluebutton-html5/imports/ui/components/whiteboard/slide-controls/component.jsx

```diff
@@ -0,0 +1,296 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './styles.scss';
+import Button from '/imports/ui/components/button/component';
+import { callServer } from '/imports/ui/services/api/index.js';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+
+export default class SlideControls extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { sliderValue: 100 };
+
+    this.nextSlide = this.nextSlide.bind(this);
```

Comment:
No @oswaldoacauan , this block of text is pointing to the place **where** you should bind your event handlers. If you need it, if course.
It doesn't say anything about always binding everything just because you can.